### PR TITLE
Component/gui

### PR DESCRIPTION
**Pull request by LazyScoundrel**
Skeleton for the GUI with main HTML files

### CODE REVIEW

**Comment by caroo7** [comment]
:+1:

File: web/gui/src/main/resources/bootstrap/js/bootstrap.js [hunk 1]
```diff
@@ -1,2006 +0,0 @@
-/*!
```
Comment (LukaszCzubernat) [inline web/gui/src/main/resources/bootstrap/js/bootstrap.js]: :+1:

File: web/gui/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -2,8 +2,17 @@
...
 <head>
     <meta charset="UTF-8">
-    <title>Robot library checker</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" href="../css/reset.css">
+    <!-- Latest compiled and minified CSS
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    -->
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (LukaszCzubernat) [inline web/gui/src/main/resources/templates/index.html:8]: :+1:

**Comment by LukaszCzubernat** [comment]
Look good.
Can be merged

**Comment by caroo7** [comment]
Nothing more from my side.

File: web/gui/src/main/resources/templates/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="../css/reset.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <title>Robot library checker</title>
+
+</head>
+
```
Comment (LIttleAncientForestKami) [inline web/gui/src/main/resources/templates/index.html]: No need for these blanks, right?

File: web/gui/src/main/resources/templates/logging.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <title>Log in to Yggdrasil Library Robot</title>
```
Comment (LIttleAncientForestKami) [inline web/gui/src/main/resources/templates/logging.html:4]: Yggdrasil?? :deciduous_tree:

File: web/gui/src/main/resources/templates/logging.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="../css/reset.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!-- Latest compiled JavaScript -->
```
Comment (LIttleAncientForestKami) [inline web/gui/src/main/resources/templates/logging.html]: Out of curiosity, why "latest compiled" and not Bootstrap?

File: web/gui/src/main/resources/templates/logging.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    </head>
+<body>
+
+<div class="logging">
```
Comment (LIttleAncientForestKami) [inline web/gui/src/main/resources/templates/logging.html:14]: How about more HTML 5 goodies?

```
<section class="loginForm">
<form name="login" action="submit" method="get" accept-charset="utf-8">
  <ul>
    <li><label for="usermail">Email</label>
    <input type="email" name="usermail" placeholder="yourname@email.com" required></li>
    <li><label for="password">Password</label>
    <input type="password" name="password" placeholder="password" required></li>
    <li>
    <input type="submit" value="Login"></li>
  </ul>
</form>
</section>
```

**Comment by LIttleAncientForestKami** [comment]
Few remarks from me - minor ones, but you may want them.

File: web/gui/src/main/resources/templates/logging.html (hunk 5 above)
Comment (LazyScoundrel) [reply web/gui/src/main/resources/templates/logging.html]: Oh it was taken from a tutorial, and they named it JS, didn't notice that :)
Changed

File: web/gui/src/main/resources/templates/logging.html (hunk 6 above)
Comment (LazyScoundrel) [reply web/gui/src/main/resources/templates/logging.html:14]: I am not sure of the usage of "for" in the label, from what I can see it will submit it with mapping usermail - provided email; password - provided password . Still IntelliJ is telling me about invalid id reference sooo I haven't got to this part yet :D What I need to provide?

**Comment by LazyScoundrel** [comment]
So can we merge it?
